packages/client-sdk-web/src/cache-client.ts

```diff
@@ -1,70 +1,32 @@
 import {ControlClient} from './internal/control-client';
 import {DataClient} from './internal/data-client';
-import {PingClient} from './internal/ping-client';
-import {
-  CredentialProvider,
-  NoopMomentoLoggerFactory,
-} from '@gomomento/sdk-core';
 import {
   AbstractCacheClient,
   IControlClient,
   ICacheClient,
   IDataClient,
-  IPingClient,
 } from '@gomomento/sdk-core/dist/src/internal/clients';
-
-export interface CacheClientProps {
-  credentialProvider: CredentialProvider;
-}
+import {CacheClientProps} from './cache-client-props';
 
 export class CacheClient extends AbstractCacheClient implements ICacheClient {
   constructor(props: CacheClientProps) {
     const controlClient: IControlClient = createControlClient(props);
     const dataClient: IDataClient = createDataClient(props);
-    const pingClient: IPingClient = createPingClient(props);
-    super(controlClient, [dataClient], pingClient);
+    super(controlClient, [dataClient]);
```

Comment:
It looks like my [ping client work](https://github.com/momentohq/client-sdk-nodejs/pull/516) got removed. Can you please add it back in here and restore the `createPingClient` function below as well? They're both likely to get moved out of this file when we work on [lazy loading the ping client](https://github.com/momentohq/client-sdk-nodejs/issues/517), but in the meantime we need to get it added back.

Reply:
I must have overwritten it when I merged that in. I'll add it back.